CHANGELOG.md

```diff
@@ -98,6 +98,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
  - Pass full RestResponse to user from Extension ([#4356](https://github.com/opensearch-project/OpenSearch/pull/4356))
  - Handle named wildcards (REST path parameters) ([#4415](https://github.com/opensearch-project/OpenSearch/pull/4415))
  - Adding ActionListener onFailure to ExtensionsOrchestrator ([#61](https://github.com/opensearch-project/opensearch-sdk/issues/61))
+ - Adding create component extension point support for AD ([#78](https://github.com/opensearch-project/opensearch-sdk-java/issues/78))
```

Comment:
should we be adding this PR (4517) here instead of the SDK issue number?

Reply:
Ill modify the changelog and add the link to this PR instead of the issue number